### DISCUSSION TITLE
Influence the outcome.

### DISCUSSION THREAD

**Discussion by Toolfolks** [souzatharsis/podcastfy#94] [Ideas]
eg create a positive or a negative review ?
make it focus on certain aspects (eg here is some text that XYZ said. Focus on showing the hypocrisy here or highlight all the positives from this?

Any help appreciated

**Comment by souzatharsis** [comment]
This has been implemented.

See https://github.com/souzatharsis/podcastfy/blob/main/usage/how-to.md#how-to-steer-the-conversation

Please let me know if you have any feedback
